### DISCUSSION TITLE
Does anyone have such an instagram redirect script?

### DISCUSSION THREAD

**Discussion by Clancy228** [community/community#122389] [Programming Help]
### Select Topic Area

Question

### Body

The user clicks on the link in the bio instagram, the script should open the link in an external browser (chrome or safari), + when he clicks back, he is redirected to the site again.

 (gif)
![saved_image](https://github.com/community/community/assets/168961220/dceec0b9-446f-483a-9a71-e508edc50ae5)

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @Clancy228 !

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the `Feed` category is solely related to conversations around the GitHub product `Feed`. This question should be in the Programming Help category. I've gone ahead and moved it for you. Good luck!

**Comment by Walter456951** [comment]
Hi! If you’re looking for an Instagram redirect script, you might want to check out tools that let you manage your instagram bio links more efficiently. These scripts are designed to open links in an external browser like Chrome or Safari, and when a user clicks back, they can be redirected to your site again. This is especially useful if you want to keep visitors engaged without losing them after they explore the link. I found this method works well for influencers, marketers, or anyone managing multiple links in their [Instagram bio](https://stylishbios.in/). Using such scripts can simplify navigation, improve user experience, and ensure your visitors easily return to your main page after checking the linked content.

**Comment by tanishrajh** [comment]
You can’t fully do this. Instagram opens links in its in-app browser, and there’s no script that can force Chrome or Safari to open automatically — this is a platform restriction.

Best practice workaround:
Detect Instagram’s in-app browser
Show a message asking users to tap ⋮ → “Open in browser”

Example:

<script>
  if (navigator.userAgent.includes("Instagram")) {
    alert("Please open this link in your browser using the menu (⋮).");
  }
</script>


Redirecting again when the user presses “back” is unreliable and often blocked, especially on iOS.

**Comment by thecatlens-prog** [comment]
If you’re a fan of emotional poetry and meaningful lines, then you should definitely check out Punjabi Shayari
. This website offers a beautiful collection of [Punjabi shayari](https://punjabishayarii.com/), sad quotes, love status, attitude lines, and heart-touching poetry for every mood. Whether you want captions for Instagram, WhatsApp status, or simply enjoy reading Punjabi poetry, this platform has something special for everyone. The content is regularly updated, easy to explore, and perfect for people who love Punjabi language and emotions expressed through words.
